build/dependenciesTest.props

```
@@ -2,12 +2,15 @@
   <PropertyGroup>
     <BannedApiAnalyzersVersion>3.3.4</BannedApiAnalyzersVersion>
     <DotNetCoreAppRuntimeVersion>2.1.30</DotNetCoreAppRuntimeVersion>
-    <MicrosoftAzureKeyVaultCryptographyVersion>2.0.5</MicrosoftAzureKeyVaultCryptographyVersion>
+    <MicrosoftAzureKeyVaultCryptographyVersion>3.0.5</MicrosoftAzureKeyVaultCryptographyVersion>
     <MicrosoftNETTestSdkVersion>17.11.0</MicrosoftNETTestSdkVersion>
+    <MicrosoftRestClientRuntime>2.3.24</MicrosoftRestClientRuntime>
     <NetStandardVersion>2.0.3</NetStandardVersion>
     <NewtonsoftVersion>13.0.3</NewtonsoftVersion>
+    <SystemNetHttp>4.3.4</SystemNetHttp>
     <SystemSecurityClaimsVersion>4.3.0</SystemSecurityClaimsVersion>
     <SystemTextJson>8.0.4</SystemTextJson>
+    <SystemTextRegularExpressions>4.3.1</SystemTextRegularExpressions>
     <XunitRunnerConsoleVersion>2.9.0</XunitRunnerConsoleVersion>
     <XunitRunnerVisualStudioVersion>2.8.2</XunitRunnerVisualStudioVersion>
```

This might  help run the tests with .NET 9 better in the release build:

```suggestion
    <XunitRunnerVisualStudioVersion>3.0.0-pre.30</XunitRunnerVisualStudioVersion>
```

---

Thanks, was going to do that in a separate PR after